<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Taxonomy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconsResource/dla.png</normaloff>:/iconsResource/dla.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QWidget" name="mainTab">
      <property name="styleSheet">
       <string notr="true">QScrollArea {
	background: transparent;
	border: transparent;
}
QScrollArea &gt; QWidget &gt; QWidget {
	background: transparent;
	border: transparent;
}
QScrollArea &gt; QWidget {
	background: transparent;
	border: transparent;
}</string>
      </property>
      <attribute name="icon">
       <iconset>
        <normaloff>:/iconsResource/prettyOffice/Ordering32.png</normaloff>:/iconsResource/prettyOffice/Ordering32.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Main Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="mainScroll">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>461</width>
            <height>338</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="mainGrid">
             <item row="5" column="0">
              <widget class="QLabel" name="labelFolder">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Folder Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelIcon">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Icon:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" rowspan="2" colspan="2">
              <widget class="QLineEdit" name="linePlural">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="lineNoun"/>
             </item>
             <item row="5" column="1" colspan="2">
              <widget class="QLineEdit" name="lineFolder"/>
             </item>
             <item row="6" column="0" colspan="3">
              <widget class="QCheckBox" name="checkVisible">
               <property name="text">
                <string>Visible</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelTable">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Table Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="2">
              <widget class="QLineEdit" name="lineTable"/>
             </item>
             <item row="0" column="1" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QToolButton" name="buttonIcon">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Change Icon</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>48</width>
                   <height>48</height>
                  </size>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelNoun">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Noun Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" rowspan="2">
              <widget class="QLabel" name="labelPlural">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Plural Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="3">
              <widget class="QCheckBox" name="checkHasChildren">
               <property name="text">
                <string>Has Children</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="3">
              <widget class="QCheckBox" name="checkTagsMode">
               <property name="text">
                <string>Tags Mode</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineNoun</tabstop>
  <tabstop>linePlural</tabstop>
  <tabstop>lineTable</tabstop>
  <tabstop>lineFolder</tabstop>
  <tabstop>checkVisible</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>mainScroll</tabstop>
 </tabstops>
 <connections/>
</ui>
